src/hw/hw_mngr.cpp

```diff
@@ -13,10 +13,108 @@
 
 #include <charconv>
 #include <filesystem>
+#include <iostream>
 
 namespace fs = std::filesystem;
 using namespace phosphor::logging;
 
+namespace {
+    constexpr int maxErrorAttempts = 20;
+    constexpr const char* dbusPathFanSensorPwm = "/xyz/openbmc_project/sensors/fan_pwm/";
+    constexpr const char* dbusPathFanSensorRpm = "/xyz/openbmc_project/sensors/fan_tach/";
+    constexpr const char* dbusFanSensorService = "xyz.openbmc_project.FanSensor";
+    constexpr const char* dbusInterfaceSensorValue = "xyz.openbmc_project.Sensor.Value";
+    constexpr const char* dbusPropertyNameValue = "Value";
+    constexpr const char* emptyString = "";
+
+
+    struct {
+        const uint32_t inletRangeMin;
+        const uint32_t inletRangeMax;
+        const uint32_t outletRangeMin;
+        const uint32_t outletRangeMax;
+        const char* partNumber;
```

Comment:
I believe we also need a different name for the fan depending on the detected P/N.
Something like "VEGMAN System Fan" and "VEGMAN HiPerf System Fan"
Check with the product team about it please.

Reply:
fixed